From 67fbb411daaaecf4bf0edd66875cbdf70d25c5a2 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Thu, 18 Oct 2018 10:29:41 +0800
Subject: [PATCH] arm64: dts: rockchip: support usb3 phy for rk1808 dwc3

The combphy which supports PCIe/USB3.0 on rk1808 has been
enabled, so we can used it as usb3-phy for DWC3 controller
by default.

Change-Id: I106885eb79621b40214bc2ebac43d8f87ac63687
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 4dc5cf36bbef..dc2952e24bb0 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -153,8 +153,8 @@
 			reg = <0x0 0xfd000000 0x0 0x200000>;
 			interrupts = <GIC_SPI 75 IRQ_TYPE_LEVEL_HIGH>;
 			dr_mode = "peripheral";
-			phys = <&u2phy_otg>;
-			phy-names = "usb2-phy";
+			phys = <&u2phy_otg>, <&combphy PHY_TYPE_USB3>;
+			phy-names = "usb2-phy", "usb3-phy";
 			phy_type = "utmi_wide";
 			snps,dis_enblslpm_quirk;
 			snps,dis-u2-freeclk-exists-quirk;
-- 
2.35.3

